models/classes/QtiResultsService.php

```diff
@@ -354,4 +356,18 @@ private function getDisplayDate(string $epoch): string
     {
         return tao_helpers_Date::displayeDate($epoch, tao_helpers_Date::FORMAT_ISO8601);
     }
+
+    /**
+     * Prepares a variable value depending on it's baseType
+     */
+    private function prepareItemVariableValue($value, $basetype): string
+    {
+        if ($basetype === 'file') {
+            /** @var BinaryContentRenderer $renderer */
+            $renderer = $this->getServiceLocator()->get(BinaryContentRenderer::class);
```

Comment:
There are no need to define default config at `config/default/BinaryContentRenderer.conf.php` if you are not passing any options to the service

Reply:
Removed

Reply:
Reverted